sympy/utilities/tests/test_code_quality.py

```diff
@@ -205,7 +205,10 @@ def test_this_file(fname, test_file):
         "setupegg.py",
     ]]
     # Files to exclude from all tests
-    exclude = set()
+    exclude = set([
```

Comment:
Is this also excluded by the master blacklist?

Reply:
I believe the blacklists are separate. This one can stay. Although if the generated file has trailing whitespace it would be nice to remove that as some people (including me) have their editors configured to remove it automatically.

Reply:
Added whitespace stripping. I tried running these through yapf, and the results were less than readable, so this is probably as much prodding as we'll want.